Allow admins to vote submissions

damianmarti opened the pull request:
Vote with a score from 1 to 10. 

Public votes for now.

![localhost_3000_admin](https://github.com/user-attachments/assets/7b9da98d-a41c-48ef-88cb-1566487fe570)

close #19

Pabl0cks commented on packages/nextjs/app/admin/_components/SubmissionCard.tsx:
```diff
@@ -53,44 +60,93 @@ export const SubmissionCard = ({ submission }: { submission: Submission }) => {
+      <div className="card bg-primary text-primary-content">
+        <div className="card-body">
+          <div className="flex mb-4 items-center">
+            <div className="rating">
+              <input type="radio" name={`rating_${submission.id}`} className="rating-hidden" checked={score === 0} />
+              {[...Array(10)].map((_e, i) => (
+                <input
+                  type="radio"
+                  name={`rating_${submission.id}`}
+                  className="mask mask-star"
+                  title={(i + 1).toString()}
+                  checked={score === i + 1}
+                  key={i}
+                  onClick={() => vote(i + 1)}
+                />
+              ))}
+            </div>
```
I'm getting warnings in the console, maybe we need to use `onChange` instead of `onClick`?

```
at async Submissions (./app/admin/_components/Submissions.tsx:16:25)
Warning: You provided a checked prop to a form field without an onChange han
dler. This will render a read-only field. If the field should be mutable use de
faultChecked. Otherwise, set either onChange or readOnly.
    at input
    at div
    at div
    at div
    at div
    at div
    at SubmissionCard (webpack-internal:///(ssr)/./app/admin/_components/Submiss
ionCard.tsx:32:27)
    at Lazy
    at div
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/not-found-boundary.js:54:9)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at main
    at div
    at ScaffoldEthApp (webpack-internal:///(ssr)/./components/ScaffoldEthAppWith
Providers.tsx:42:27)
    at div
    at ModalProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainb
owkit/dist/index.js:7949:26)
    at ShowBalanceProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me
/rainbowkit/dist/index.js:3312:32)
    at TransactionStoreProvider (webpack-internal:///(ssr)/./node_modules/@rainb
ow-me/rainbowkit/dist/index.js:3780:37)
    at ModalSizeProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/r
ainbowkit/dist/index.js:3959:30)
    at I18nProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainbo
wkit/dist/index.js:2961:23)
    at WalletButtonProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-m
e/rainbowkit/dist/index.js:3942:33)
    at RainbowKitChainProvider (webpack-internal:///(ssr)/./node_modules/@rainbo
w-me/rainbowkit/dist/index.js:3280:36)
    at RainbowKitProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/
rainbowkit/dist/index.js:4658:31)
    at RainbowKitAuthenticationProvider (webpack-internal:///(ssr)/./node_module
s/@rainbow-me/rainbowkit/dist/index.js:2201:45)
    at RainbowKitSiweNextAuthProvider (webpack-internal:///(ssr)/./node_modules/
@rainbow-me/rainbowkit-siwe-next-auth/dist/index.js:14:43)
    at QueryClientProvider (webpack-internal:///(ssr)/./node_modules/@tanstack/r
eact-query/build/modern/QueryClientProvider.js:23:30)
    at SessionProvider (webpack-internal:///(ssr)/./node_modules/next-auth/react
/index.js:429:26)
    at Hydrate (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/hydrate.
js:10:13)
    at WagmiProvider (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/co
ntext.js:12:13)
    at ScaffoldEthAppWithProviders (webpack-internal:///(ssr)/./components/Scaff
oldEthAppWithProviders.tsx:88:40)
    at Lazy
    at f (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:21:24)
    at $ (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:18:62)
    at ThemeProvider (webpack-internal:///(ssr)/./components/ThemeProvider.tsx:1
3:26)
    at Lazy
    at body
    at html
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at ReactDevOverlay (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/react-dev-overlay/internal/ReactDevOverlay.js:66:9)
    at HotReload (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/react-dev-overlay/hot-reloader-client.js:296:11)
    at Router (webpack-internal:///(ssr)/./node_modules/next/dist/client/compone
nts/app-router.js:155:11)
    at ErrorBoundaryHandler (webpack-internal:///(ssr)/./node_modules/next/dist/
client/components/error-boundary.js:99:9)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at AppRouter (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/app-router.js:421:13)
    at Lazy
    at Lazy
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at ServerInsertedHTMLProvider (C:\GitHub\extensions-hackathon\packages\nextj
s\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:17605)
Warning: You provided a checked prop to a form field without an onChange han
dler. This will render a read-only field. If the field should be mutable use de
faultChecked. Otherwise, set either onChange or readOnly.
    at input
    at div
    at div
    at div
    at div
    at div
    at SubmissionCard (webpack-internal:///(ssr)/./app/admin/_components/Submiss
ionCard.tsx:32:27)
    at Lazy
    at div
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/not-found-boundary.js:54:9)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at main
    at div
    at ScaffoldEthApp (webpack-internal:///(ssr)/./components/ScaffoldEthAppWith
Providers.tsx:42:27)
    at div
    at ModalProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainb
owkit/dist/index.js:7949:26)
    at ShowBalanceProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me
/rainbowkit/dist/index.js:3312:32)
    at TransactionStoreProvider (webpack-internal:///(ssr)/./node_modules/@rainb
ow-me/rainbowkit/dist/index.js:3780:37)
    at ModalSizeProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/r
ainbowkit/dist/index.js:3959:30)
    at I18nProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainbo
wkit/dist/index.js:2961:23)
    at WalletButtonProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-m
e/rainbowkit/dist/index.js:3942:33)
    at RainbowKitChainProvider (webpack-internal:///(ssr)/./node_modules/@rainbo
w-me/rainbowkit/dist/index.js:3280:36)
    at RainbowKitProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/
rainbowkit/dist/index.js:4658:31)
    at RainbowKitAuthenticationProvider (webpack-internal:///(ssr)/./node_module
s/@rainbow-me/rainbowkit/dist/index.js:2201:45)
    at RainbowKitSiweNextAuthProvider (webpack-internal:///(ssr)/./node_modules/
@rainbow-me/rainbowkit-siwe-next-auth/dist/index.js:14:43)
    at QueryClientProvider (webpack-internal:///(ssr)/./node_modules/@tanstack/r
eact-query/build/modern/QueryClientProvider.js:23:30)
    at SessionProvider (webpack-internal:///(ssr)/./node_modules/next-auth/react
/index.js:429:26)
    at Hydrate (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/hydrate.
js:10:13)
    at WagmiProvider (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/co
ntext.js:12:13)
    at ScaffoldEthAppWithProviders (webpack-internal:///(ssr)/./components/Scaff
oldEthAppWithProviders.tsx:88:40)
    at Lazy
    at f (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:21:24)
    at $ (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:18:62)
    at ThemeProvider (webpack-internal:///(ssr)/./components/ThemeProvider.tsx:1
3:26)
    at Lazy
    at body
    at html
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at ReactDevOverlay (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/react-dev-overlay/internal/ReactDevOverlay.js:66:9)
    at HotReload (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/react-dev-overlay/hot-reloader-client.js:296:11)
    at Router (webpack-internal:///(ssr)/./node_modules/next/dist/client/compone
nts/app-router.js:155:11)
    at ErrorBoundaryHandler (webpack-internal:///(ssr)/./node_modules/next/dist/
client/components/error-boundary.js:99:9)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at AppRouter (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/app-router.js:421:13)
    at Lazy
    at Lazy
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at ServerInsertedHTMLProvider (C:\GitHub\extensions-hackathon\packages\nextj
s\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:17605)
Warning: You provided a checked prop to a form field without an onChange han
dler. This will render a read-only field. If the field should be mutable use de
faultChecked. Otherwise, set either onChange or readOnly.
    at input
    at div
    at div
    at div
    at div
    at div
    at SubmissionCard (webpack-internal:///(ssr)/./app/admin/_components/Submiss
ionCard.tsx:32:27)
    at Lazy
    at div
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/not-found-boundary.js:54:9)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at main
    at div
    at ScaffoldEthApp (webpack-internal:///(ssr)/./components/ScaffoldEthAppWith
Providers.tsx:42:27)
    at div
    at ModalProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainb
owkit/dist/index.js:7949:26)
    at ShowBalanceProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me
/rainbowkit/dist/index.js:3312:32)
    at TransactionStoreProvider (webpack-internal:///(ssr)/./node_modules/@rainb
ow-me/rainbowkit/dist/index.js:3780:37)
    at ModalSizeProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/r
ainbowkit/dist/index.js:3959:30)
    at I18nProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainbo
wkit/dist/index.js:2961:23)
    at WalletButtonProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-m
e/rainbowkit/dist/index.js:3942:33)
    at RainbowKitChainProvider (webpack-internal:///(ssr)/./node_modules/@rainbo
w-me/rainbowkit/dist/index.js:3280:36)
    at RainbowKitProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/
rainbowkit/dist/index.js:4658:31)
    at RainbowKitAuthenticationProvider (webpack-internal:///(ssr)/./node_module
s/@rainbow-me/rainbowkit/dist/index.js:2201:45)
    at RainbowKitSiweNextAuthProvider (webpack-internal:///(ssr)/./node_modules/
@rainbow-me/rainbowkit-siwe-next-auth/dist/index.js:14:43)
    at QueryClientProvider (webpack-internal:///(ssr)/./node_modules/@tanstack/r
eact-query/build/modern/QueryClientProvider.js:23:30)
    at SessionProvider (webpack-internal:///(ssr)/./node_modules/next-auth/react
/index.js:429:26)
    at Hydrate (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/hydrate.
js:10:13)
    at WagmiProvider (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/co
ntext.js:12:13)
    at ScaffoldEthAppWithProviders (webpack-internal:///(ssr)/./components/Scaff
oldEthAppWithProviders.tsx:88:40)
    at Lazy
    at f (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:21:24)
    at $ (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:18:62)
    at ThemeProvider (webpack-internal:///(ssr)/./components/ThemeProvider.tsx:1
3:26)
    at Lazy
    at body
    at html
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at ReactDevOverlay (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/react-dev-overlay/internal/ReactDevOverlay.js:66:9)
    at HotReload (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/react-dev-overlay/hot-reloader-client.js:296:11)
    at Router (webpack-internal:///(ssr)/./node_modules/next/dist/client/compone
nts/app-router.js:155:11)
```

Pabl0cks reviewed (commented):
Nice job Damu! Voting system is simple and does the job 🙌

I left a comment with a warning I was getting. 

=============================================

I think we could tweak the UI/UX a bit but we can tackle it later, and we could ask help to @andrealbiac for it. Leaving a couple of notes I took about it:

- We could show more than 1 submission per row
- We should force cards to have the same weight
- Comments could be more compact and with collapse icon

Leaving some screenshots here too.

<details><summary>Cards without same weight</summary>

![voting-cards](https://github.com/user-attachments/assets/1f702d2a-8b4b-40cc-be35-48feca6a1f46)
</details>
<details><summary>Current comments</summary>

![voting-comments](https://github.com/user-attachments/assets/aa904a0a-4faf-41a3-be41-fa37daafba15)
</details>
<details><summary>Compact comments idea</summary>

![voting-comments-proposal](https://github.com/user-attachments/assets/8d666d26-a024-413b-8e65-f266d1fb9825)
</details>

Pabl0cks commented:
I like the 1 to 10 system better than 0 to 10 commented in https://github.com/BuidlGuidl/extensions-hackathon/issues/9 because you clearly see which ones you voted already, and which ones are missing your vote.

Once you vote you cant undo the vote and leave it empty, but I don't see any problem.

damianmarti commented:
> Nice job Damu! Voting system is simple and does the job 🙌
> 
> I left a comment with a warning I was getting.
> 
> =============================================
> 
> I think we could tweak the UI/UX a bit but we can tackle it later, and we could ask help to @andrealbiac for it. Leaving a couple of notes I took about it:
> 
> * We could show more than 1 submission per row
> * We should force cards to have the same weight
> * Comments could be more compact and with collapse icon
> 
> Leaving some screenshots here too.
> 
> Cards without same weight
> Current comments
> Compact comments idea

Thanks for the review! 

Yes, we need to improve how this looks for sure!

damianmarti commented:
@Pabl0cks I fixed the warning and I added the option to remove a vote (maybe the user made a mistake) adding an "x" to the left of the stars.

![localhost_3000_admin (1)](https://github.com/user-attachments/assets/0be46524-c94f-4abd-b01e-14cf3c68281c)

Pabl0cks reviewed (approved):
LGTM as a first iteration of voting!! 

Tweaked the style for vote deletion a bit:
![image](https://github.com/user-attachments/assets/7e93d485-129a-4877-8f15-f290faf39e94)

Will create an issue to upgrade the design and responsive 🙌

carletex commented:
Updating the live database! SInce we don't have a migration workflow I have to do it manually everytime we change the database.
